compile.sh

```diff
@@ -18,6 +19,7 @@ set -o errtrace
 working_dir="$(pwd)"
 log_dir="${working_dir}"/logs
 out_dir="${working_dir}"/out # Directory for generated ISOs
+docker_repo="remisa/anarchy:latest"
```

Comment:
I'm not sure how I feel about always downloading from dockerhub (and one of us having to push every update to dockerhub) compared to locally executing the dockerfile.

What are the pros of downloading from dockerhub?

Reply:
I prefer myself and recommend everyone build his/her own image locally but still pushed the image to hub as an extra option to users. Feel free to change/remove any part of it.
I have tried to leave scripts as much as possible outside the image (even `docker.compile.sh` is not embedded into image) so for development purposes an old image continues to work for months.

> one of us having to push every update to dockerhub

That can be automated using hooks and docker supports it but I have no prior experience with it.

Still cannot figure out on myself how the image size can be reduced and make it completely offline (container still downloads some packages on every build) so any help is appreciated.

Reply:
> Still cannot figure out on myself how the image size can be reduced and make it completely offline (container still downloads some packages on every build) so any help is appreciated.

I don't mind it needing internet access to download the arch:latest part, but since I don't have much experience with Docker, I don't understand why it needs to download `remisa/anarchy:latest` instead of just building the image locally.